### PR TITLE
Warn on packages missing from PyPi instead of crashing

### PR DESCRIPTION
**Pull request by FiloSottile**

### CODE REVIEW

**Comment by rickard-von-essen-iz** [comment]
👍

File: poet/poet.py [hunk 1]
```diff
@@ -166,8 +169,14 @@ def make_graph(pkg):
             dependencies[package]['version'] = None
 
     for package in dependencies:
-        package_data = research_package(package, dependencies[package]['version'])
-        dependencies[package].update(package_data)
+        try:
+            package_data = research_package(package, dependencies[package]['version'])
+            dependencies[package].update(package_data)
+        except:
```
Comment (tdsmith) [inline poet/poet.py]: the naked except here seems inhygenic :)

**Comment by Kriechi** [comment]
Bump!

Do you need help getting this merged? 
I'm happy to send a new PR with proper exception handling, if this is the only thing holding this PR back...

**Comment by tdsmith** [comment]
I'd be happy to review that.

**Pull request closed without merging**